lib/actions/form.js

```diff
@@ -133,7 +101,7 @@ export function formChanged (oldQuery, newQuery) {
     }
 
     // Check whether a trip should be auto-replanned
-    const {autoPlan, debouncePlanTimeMs} = otpState.config
+    const { autoPlan, debouncePlanTimeMs } = otpState.config
```

Comment:
oh man, I prefer the other way. I think since Trevor gave us full control over mastarm we oughta update the eslint config to have one bracket spacing rule.

Reply:
Yea, I go back and forth on this rule, but have just been doing whatever the linter tells me to do lately, haha.

Reply:
What linting rules are you using? I don't see [space-in-brackets](https://eslint.org/docs/rules/space-in-brackets) defined in [mastarm's eslintrc.json](https://github.com/conveyal/mastarm/blob/dev/lib/eslintrc.json) file.

Reply:
I think I'm just using [standard](https://standardjs.com/).